src/org/loklak/harvester/RedirectUnshortener.java

```diff
@@ -57,7 +57,7 @@
         "flip.it",
         "lnkd.in"
     };
```

Comment:
Is the string of working hosts ever going to get updated on a successful unshortening?